onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphPartitioner.cpp

```diff
@@ -219,11 +219,14 @@ namespace Dml
 
     bool DoesNodeContainSupportedDataTypes(
         const onnxruntime::Node& node,
-        const std::unordered_map<std::string, GraphPartition*>& nodeNameToPartitionMap,
+        bool allow64BitInputThroughStrides,
```

Comment:
>allow64BitInputThroughStrides [](start = 13, length = 29)

o_O
Oh boy, this new parameter rather complicates already complex code. I'm looking forward to hopefully deleting it later with [https://microsoft.visualstudio.com/OS/_workitems/edit/24643526](https://microsoft.visualstudio.com/OS/_workitems/edit/24643526).